### ISSUE TITLE
Blood pressure reduction improves symptoms

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9926]
**Preferred term label:**
Blood pressure reduction improves symptoms

**Synonyms**
Improvement with hypertension control, lowering blood pressure improves symptoms, BP control, BP reduction

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Posterior reversible encephalopathy syndrome, an illness in which a person can present with acutely altered mentation, drowsiness or sometimes stupor, visual impairment (e.g., visual hallucinations, cortical blindness, hemianopia, quadrantanopia, and diplopia), seizures (focal or general tonic-clonic), and sudden or constant, non-localized headaches.

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I think this is probably not a great HPO term because hypertension control will improve many symptoms depending on the context and so this is unlikely to be discriminatory except in very specific situations.

**Comment by MickeySegal** [comment]
We now include many such conditions, including aortic dissection, aortic regurgitation and mitral regurgitation.
